src/progress-indicator/progress-indicator.component.spec.ts

```diff
@@ -7,9 +7,14 @@ import { CommonModule } from "@angular/common";
 import { DialogModule, ExperimentalModule } from "..";
 import { CheckmarkOutline16Module } from "@carbon/icons-angular/lib/checkmark--outline/16";
 import { Warning16Module } from "@carbon/icons-angular/lib/warning/16";
+import { Step } from "./progress-indicator-step.interface";
 
 @Component({
-	template: `<ibm-progress-indicator [steps]="steps" [current]="current"></ibm-progress-indicator>`
+	template: `<ibm-progress-indicator
+					[steps]="steps"
+					[current]="current"
+					(onStepSelected)="onStepSelected.emit($event)">
```

Comment:
```suggestion
					(stepSelected)="onStepSelected.emit($event)">
```

Reply:
done...i also added the action in the story